### DISCUSSION TITLE
Export/save as inline css

### DISCUSSION THREAD

**Discussion by kerepuki** [givanz/VvvebJs#371] [General]
I love this library. You've done an amazing job. I'd like to use multiple versions. One for a full webpage editor and another for email campaigns. As emails required inline styles, would it be possible to create a version to be able to export/save the html as inline?

I realise I could use an inliner plugin but it doesn't support columns.

Also, it would be great to simply be able to turn off elements that might not be supported in emails.

**Comment by givanz** [comment]
Thank you.

With the latest update https://github.com/givanz/VvvebJs/commit/ef55737e576b4cbd506e77621401bcda19d0f6ed you can enable inline css with  https://github.com/givanz/VvvebJs/blob/master/libs/builder/builder.js#L2645

```javascript
Vvveb.StyleManager.inlineCSS = true;
```

By elements you mean components? 

You can change what components are loaded by redefining the components group.

For example for Bootstrap 5 group https://github.com/givanz/VvvebJs/blob/master/libs/builder/components-bootstrap5.js#L19-L20 you can add the following code to limit what components will be loaded.

```javascript
if (email)  {
	Vvveb.ComponentsGroup['Bootstrap 5'] =
	["html/btn", "html/btn-link", "html/card"];
}
```


before initialization https://github.com/givanz/VvvebJs/blob/master/editor.html#L2245

**Comment by hayden-t** [comment]
Ive found MJML the best for email templating as it accounts for all the idiosyncrasies of different email client renderers
